pkg/common/cns-lib/volume/manager.go

```diff
@@ -172,6 +175,7 @@ type ExpandVolumeExtraParams struct {
 	StorageClassName                     string
 	StoragePolicyID                      string
 	Namespace                            string
+	Capacity                             *resource.Quantity
```

Comment:
Can you add comment to explain what is this size?  This is especially important in the context of volume expansion as we have original and new sizes.

Reply:
Fixed.